HackYourFuture-CPH/fp-class20
# Update folder structure client

# Description
- Adding `assets` folder in `packages/client/public/` so that we can store all the images related to the Simply Spices project
- Moving the Button Component, Story, and CSS to it's own folder so that we organise the components this way - `packages/client/src/components/folder-component` which this case folder-component is `Button`

Fixes https://hackyourfuture-dk.atlassian.net/browse/CLASS20-22?atlOrigin=eyJpIjoiYmZiN2IyYmY5MjNiNDliYjg0MjQwYTA3N2I1NzAyNmMiLCJwIjoiaiJ9

# How to test?

- To test the client you can run `yarn workspace client start `on the root of the project - it should open a browser with http://localhost:3000/
- To test storybook you can run `yarn workspace client storybook` on the root of the project - it should open a browser with http://localhost:6006/?path=/story/example-button--small

# Checklist
